docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - /var/lib/postgresql
 
   hmpps-auth:
-    image: quay.io/hmpps/hmpps-auth:latest
+    image: quay.io/hmpps/hmpps-auth::2024-05-09.23133.61d6975
```

Comment:
`quay.io/hmpps/hmpps-auth::2024-05-09.23133.61d6975`
appears to have an extra `:`

Reply:
oops. thanks 😄 

```suggestion
    image: quay.io/hmpps/hmpps-auth:2024-05-09.23133.61d6975
```